tests/testthat/_snaps/after-wrappers.md

```diff
@@ -660,62 +660,33 @@
-      [136] "reverse"           "rle"               "rle_id"           
-      [139] "rolling_max"       "rolling_mean"      "rolling_median"   
-      [142] "rolling_min"       "rolling_quantile"  "rolling_skew"     
-      [145] "rolling_std"       "rolling_sum"       "rolling_var"      
-      [148] "round"             "sample"            "search_sorted"    
-      [151] "set_sorted"        "shape"             "shift"            
-      [154] "shift_and_fill"    "shrink_dtype"      "shuffle"          
-      [157] "sign"              "sin"               "sinh"             
-      [160] "skew"              "slice"             "sort"             
-      [163] "sort_by"           "sqrt"              "std"              
-      [166] "str"               "struct"            "sub"              
-      [169] "sum"               "tail"              "tan"              
-      [172] "tanh"              "to_frame"          "to_list"          
-      [175] "to_lit"            "to_physical"       "to_r"             
-      [178] "to_struct"         "to_vector"         "top_k"            
-      [181] "unique"            "unique_counts"     "upper_bound"      
-      [184] "value_counts"      "var"               "xor"              
-
----
-
-    Code
-      ls(.pr[[private_key]])
-    Output
-       [1] "add"                         "alias"                      
-       [3] "all"                         "any"                        
-       [5] "append_mut"                  "arg_max"                    
-       [7] "arg_min"                     "chunk_lengths"              
-       [9] "clone"                       "compare"                    
-      [11] "div"                         "dtype"                      
-      [13] "equals"                      "fast_explode_flag"          
-      [15] "from_arrow_array_robj"       "from_arrow_array_stream_str"
-      [17] "get_fmt"                     "is_sorted"                  
-      [19] "is_sorted_flag"              "is_sorted_reverse_flag"     
-      [21] "len"                         "map_elements"               
-      [23] "max"                         "mean"                       
-      [25] "median"                      "min"                        
-      [27] "mul"                         "n_unique"                   
-      [29] "name"                        "new"                        
-      [31] "panic"                       "print"                      
-      [33] "rem"                         "rename_mut"                 
-      [35] "rep"                         "set_sorted_mut"             
-      [37] "shape"                       "sleep"                      
-      [39] "sort_mut"                    "std"                        
-      [41] "sub"                         "sum"                        
-      [43] "to_fmt_char"                 "to_frame"                   
-      [45] "to_r"                        "value_counts"               
-      [47] "var"                        
```

Comment:
These lines have been removed.
Please run the snapshot test again.

Reply:
Thank you for your patient guidance.